AM_HW9_Notebook.ipynb

```diff
@@ -0,0 +1,461 @@
+{
```

Comment:
<p>Interesting velocity you want to use. Did you think about in what reference frame you measure your acceleration? Does it include eg the rotation of earth?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1_Calculus/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>